php/tower/composer.json

```
@@ -0,0 +1,39 @@
+{
+    "name": "tower/tower",
+    "description": "The Tower Framework",
+    "keywords": ["framework", "tower" , "high performance"],
+    "type": "project",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Amir",
+            "email": "faramarzii.amir@gmail.com"
+        }
+    ],
+    "require": {
+        "php" : ">=8.0",
+        "tower/tower-framework": "^1.0"
```

```suggestion
        "tower/tower-framework": "~1.11.0"
```

using `^1.0` could lead to `1.1` which will made doc out of sync

PS : version will be updated by `renovatebot`